MIGRATION.md

```diff
@@ -11,10 +11,22 @@ There are several ways to create a client connection in `cloudant-node-sdk`:
 
 ## Other differences
 1. Using the `dotenv` package to store credentials in a file is not recommended. See the [external file configuration section](https://github.com/IBM/cloudant-node-sdk#authentication-with-external-configuration) in our API docs for handling this feature in our new library.
-1. Fetching the database object first before performing additional operations is not required. For example, in the case of updating a document you would first call `getDocument` to fetch and then `putDocument` to update.
+1.  In `cloudant-node-sdk` all operations are performed from the scope of the client instance and
+    not associated with any sub-scope like the database. There is no need to instantiate a
+    database object to interact with documents - the database name is included as part of
+    document operations. For example, in the case of updating a document you would first call
+    `getDocument` to fetch and then `putDocument` to update. As a result of which the `use`
+    operation also became invalid.
```

Comment:
```suggestion
1.  In `cloudant-node-sdk` all request operation performed through a service client instance. 
    There is no need to instantiate a
    database object to interact with documents - the database name is included in the operation request as parameter.
     For example, in the case of updating a document you would first call
    `getDocument({ db: "dbname",docId: 'FishStew'})` to fetch and then `putDocument({ db: "dbname",docId: 'FishStew'})` to update. As a result of which the `use`
    operation also became obsoleted.
```

Reply:
I tried to align this phrase with our Java migration guide similar phrase, so I'd like to keep the first 2 sentences as is. However, I like the idea of highlighting the `db` required param.

Wdyt of:
```suggestion
1.  In `cloudant-node-sdk` all operations are performed from the scope of the client instance and
    not associated with any sub-scope like the database. 
    There is no need to instantiate a database object to interact with documents - the database name is included as part of
    document operations. For example, in the case of updating a document you would first call
    `` getDocument({ db: `${dbName}`, docId:`${docId}`}) `` to fetch and then `` putDocument({ db: `${dbName}`, docId:`${docId}`}) `` to update. As a result of which the `use`
    operation also became obsoleted.
```

Reply:
Okay, if you prefer that way. Just `scope of the client instance` and also `sub-scope like the database` for me just sounds weird. The scope is used mainly with tasks and projects...

Reply:
I looked for the original suggestion: https://github.com/cloudant/java-cloudant/pull/518#discussion_r571911928

Reply:
4a390c6